param/__init__.py

```diff
@@ -2180,6 +2180,26 @@ def __init__(self, default=Undefined, bounds=Undefined, softbounds=Undefined,
     def _validate(self, val):
         super(Range, self)._validate(val)
         self._validate_bounds(val, self.bounds, self.inclusive_bounds)
+        self._validate_step(val, self.step)
+        self._validate_order(self.bounds, allow_None=True)
```

Comment:
If the intention was to validate the order of `bounds`, then I would suggest to code that in `_validate_bounds`. As it is in this PR `Range(bounds=(1,0))` will indeed raise an error but that is going to be a bit confusing as it doesn't say that it's because of `bounds`: `ValueError: Range parameter end 0 is less than its start 1.`.